js/freewheel.js

```diff
@@ -464,28 +461,6 @@ OO.Ads.manager((function(_, $) {
     }, this);
 
     /**
-     * Called by the ad manager controller.  Pauses the current ad.
-     * @public
-     * @method Freewheel#pauseAd
-     * @param {object} ad The ad to pause
-     */
-    this.pauseAd = function(ad) {
-      OO.log("FW: Pausing the ad");
-      amc.ui.adVideoElement[0].pause();
-    };
-
-    /**
-     * Called by the ad manager controller.  Resumes the current ad.
-     * @public
-     * @method Freewheel#resumeAd
-     * @param {object} ad The ad to resume
-     */
-    this.resumeAd = function(ad) {
-      OO.log("FW: Resuming the ad");
-      amc.ui.adVideoElement[0].play();
-    };
-
-    /**
```

Comment:
Yeah, I'm conflicted about this.  You're right that they should be consistent.  I had them in there but I thought it might be confusing since I wasn't using them.  I'm thinking I should remove them from VAST as well - what do you think?